gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -80,6 +83,8 @@ public String load(String key) throws Exception {
   protected Orchestrator orchestrator;
   protected boolean isMultiActiveSchedulerEnabled;
   protected FlowCatalog flowCatalog;
+  private DagTaskStream dagTaskStream;
```

Comment:
this change monitor shouldn't use a task stream as such (that's exclusively for the `DagProcessingEngine`.)  instead let it write to the `DagManagement` API